panel/src/components/Forms/Writer/Writer.vue

```diff
@@ -65,7 +65,10 @@ export const props = {
 			})
 		},
 		extensions: Array,
-		headings: [Array, Boolean],
+		headings: {
+			default: () => [1, 2, 3, 4, 5, 6],
```

Comment:
Without a default for the headings levels, the writer won't work at all if not used in a field.